src/xpcc/architecture/platform/board/al_avreb_can/al_avreb_can.hpp

```diff
@@ -0,0 +1,69 @@
+/* Copyright (c) 2016, Roboterclub Aachen e.V.
+ * All Rights Reserved.
+ *
+ * The file is part of the xpcc library and is released under the 3-clause BSD
+ * license. See the file `LICENSE` for the full license governing this code.
+ * ------------------------------------------------------------------------ */
+
+#ifndef XPCC_AL_AVREB_CAN_HPP
+#define XPCC_AL_AVREB_CAN_HPP
+
+#include <xpcc/architecture/platform.hpp>
+#include <xpcc/debug/logger.hpp>
+
+using namespace xpcc::at90;
+
+namespace Board
```

Comment:
Could you please add more detailed information about the board, e.g. including a link to the [documentation](https://www.alvidi.de/data_sheets/pdf/products/avreb/avr_modul_avreb_english.pdf)?